docs/contributing/package-dependencies-and-auto-updates.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Package Dependencies and Auto-Updates
+description: Understanding the tscircuit package ecosystem and automated dependency update workflow
+---
+
+# Package Dependencies and Auto-Updates
+
+The tscircuit ecosystem consists of multiple interconnected packages that automatically update each other when changes are merged. This guide explains the dependency chain and how the auto-update workflow functions.
+
+## Package Dependency Flow
+
+The following diagram shows how packages in the tscircuit ecosystem depend on and trigger updates to each other:
+
+```mermaid
+flowchart TD
+
+    %% Core to Eval
+    core[tscircuit/core] -->|merge| eval[tscircuit/eval]
+
+    %% Eval to Runframe
+    eval -->|merge| runframe[tscircuit/runframe]
```

Comment:
i would remove the |merge| text, it's very misleading because it's actually a dispatch, but i think in general no text is needed